app/views/static/agreement.html.erb

```diff
@@ -11,7 +11,7 @@ When you submit your file(s) you will have the option of selecting its degree of
 
 <h3>Visibility Level: <span class="label label-success">OPEN ACCESS</span></h3>
 <p>
-By choosing the Open Access option you are allowing anyone in the world to view your submission.   You may choose to apply one of several Creative Commons licenses to your submission, which are explained on the submission form and in the FAQ.  These licenses allow you to set your own terms of use for anyone accessing the submission.  You authorize <%= t('sufia.institution_name') %> to distribute your content under the terms of that Creative Commons license and to allow users of <%= t('sufia.product_name') %> to do the same if your license choice permits that.
+By choosing the Open Access option you are allowing anyone in the world to view your submission.   You may choose to apply one of several Creative Commons licenses to your submission, which are explained on the submission form and in the FAQ.  These licenses allow you to set your own terms of use for anyone accessing the submission.  You authorize <%= t('sufia.institution_name') %> to distribute your content under the terms of that Creative Commons license and to allow users of <%= application_name %> to do the same if your license choice permits that.
 
 If you do not choose a license, you grant to <%= t('sufia.institution_name') %> the non-exclusive right to distribute your submission (including the metadata and abstract) worldwide, in any format or medium for non-commercial, research, educational, or related academic purposes only.  You also authorize <%= t('sufia.institution_name') %> to allow users of the repository to distribute your submission (including the metadata and abstract) worldwide, in any format or medium for non-commercial, research, educational, or related academic purposes only. <%= t('sufia.institution_name') %> will clearly identify your name(s) as the author(s) or owner(s) of the submission, and will not make any alteration, other than as allowed by this license, to your submission.
```

Comment:
Indeed, we should. I didn't necessarily mean to include the helper method yet, but might as well go for it.